core/src/JsonLogic/Json.hs

```diff
@@ -145,6 +145,9 @@ parseFloat (JsonString s) = fromMaybe notANumber $ readMaybe $ dropAfterSecondPo
       (l, _) -> l
 -- For an array always take the first element.
 parseFloat (JsonArray (a : _)) = parseFloat a
+-- Booleans are either 1 or 0.
+parseFloat (JsonBool True) = 1
+parseFloat (JsonBool False) = 0
```

Comment:
But according to JsonLogic {">":[true,false]} is true. So I think that according to JsonLogic true can be represented as 1.

Reply:
`>` uses a different function than parseFloat.
see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Less_than 
so use a different function to check inequality instead of changing this one.